app/javascript/listings/components/MessageModal.jsx

```diff
@@ -80,7 +86,7 @@ MessageModal.propTypes = {
     user_id: PropTypes.number.isRequired,
   }).isRequired,
   onSubmit: PropTypes.func.isRequired,
-  onChangeDraftingMessage: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
```

Comment:
Rather than changing it here I could have renamed the prop in the parent too.

Reply:
I think it may make more sense to keep the more explicit prop name `onChangeDraftingMessage` as `onChange` is pretty generic and it's not clear what it might be for.